### ISSUE TITLE
Manipulate bounds

### ISSUE THREAD

**Issue by sebastialonso** [AllanBishop/angular-img-cropper#14]
This is a great directive. Thanks you very much Allan!

I have a question though, is it possible to programatically set the bounds for the crop area?
Say I will want an 800px width crop, so when the user moves the box, I can calculate the bounds required to obtain my 800px width image, and update the bounding box.

Is this currently possible? I'm been playing around but I haven't been able to achieve this.

Thanks!

**Comment by AllanBishop** [comment]
Hi thanks!

When you say crop an area of 800px does that mean 800px in image source dimensions? So if an image was 1280px wide that was loaded, the bounds would update so that 800px of the image dimensions was covered?

**Comment by sebastialonso** [comment]
Yes, precisely.

**Comment by AllanBishop** [comment]
Yeah it is not possible at the moment but I am working on using binding to set a crop bounds width and crop bounds height so it can then be set in your controller. Also plan on passing a parameter that switches between image dimensions and screen dimensions.

**Comment by sebastialonso** [comment]
Thanks, great man. Looking forward to it!

**Comment by AllanBishop** [comment]
Just an update on this as I have been busy with work. Still going to get this done. I made an attempt a few weeks back but was not happy with how it worked but have a few ideas now.
